api/v1alpha1/baseboardmanagement_types.go

```diff
@@ -25,6 +25,7 @@ import (
 type PowerState string
 
 // BootDevice represents boot device of the BaseboardManagement.
+// +kubebuilder:validation:Enum=PXE;Disk;BIOS;CDROM;Safe
```

Comment:
Is this too validation too strict? What if there are multiple NIC's, which one gets PXE booted, or same for Disks?

Reply:
I actually had these values to the currently available options on bmclib.

I think we have not worked out the scenario for multiple NICs and disks, would that be a separate BaseboardManagement object? Since it would have different IPs etc.

Reply:
I think it depends. We'd need to uniquely identify the device somehow and that identification would be dictated by the BMC interface spec, I'd imagine. I.e. I don't know that we know what data is required to model the right thing.

Reply:
Yea I think we would need to scope that out a bit more. 
I could remove the validation for now so we don't limit ourselves 👍

Reply:
Yea, lets remove this validation right now.